src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: $duration-time;
```

Comment:
transition-duration is commonly used together with transition-property 
or maybe you might use transition shortcut as you used in catalog :)